client/src/components/Masthead/Masthead.vue

```diff
@@ -199,6 +197,7 @@ onMounted(() => {
         font-family: Verdana, sans-serif;
         font-size: 1rem;
         line-height: 2rem;
+        background: var(--masthead-color);
```

Comment:
This breaks backgrounds which do not have a simple color. E.g: image backgrounds, gradients, etc.

Reply:
I had to add this, since the brand text is otherwise not legible i.e. the custom background conflicts with the text color e.g. as shown here: 
<img width="280" alt="image" src="https://user-images.githubusercontent.com/2105447/211386455-b6466951-c014-4912-ba13-329d62295c59.png">

Reply:
The themes here are just examples, not defaults. In cases like this the theme would of course need to be adjusted for the instance. Using the background prop here makes image and gradient backgrounds not work properly for instances which have a brand text.